validate/validate.go

```diff
@@ -115,19 +115,38 @@ func (v *Validator) CheckAll() (msgs []string) {
 func (v *Validator) CheckRootfsPath() (msgs []string) {
 	logrus.Debugf("check rootfs path")
 
+	absBundlePath, err := filepath.Abs(v.bundlePath)
+	if err != nil {
+		msgs = append(msgs, fmt.Sprintf("unable to convert %q to an absolute path", v.bundlePath))
+	}
+	absBundlePath = filepath.Clean(absBundlePath)
```

Comment:
Thanks :).  Addressed this (and your other `Abs` / `Clean` points) with 8d697e8 → 899afea.